src/spdx_tools/spdx3/writer/json_ld/context.json

```diff
@@ -183,7 +193,7 @@
     },
     "createdBy": {
         "@id": "core:createdBy",
-        "@type": "core:Entity"
+        "@type": "@id"
```

Comment:
To my knowledge, the type is still `core:Agent`. If there is only a string/URI present then it will be taken as a reference to the object with the corresponding URI, which should then be of type `core:Agent`.
(also applies to all other occurrences like this)

Reply:
That would have been great, but unfortunately not what it did. Instead, it interpreted the URI as a literal and then complained that it did not have the expected type. Only by setting this to `@id` it works as expected. Shacl will still validate that the entity pointed to has the expected type due to the `sh:class` constraint.